src/pages/nft/class/_classId/index.vue

```diff
@@ -1166,6 +1205,47 @@ export default {
         1
       );
     },
+    handleSwitchToCollection(boolean) {
+      if (boolean) {
+        logTrackerEvent(
+          this,
+          'NFT',
+          'nft_class_details_switch_to_collection',
+          this.classId,
+          1
+        );
+        this.shouldShowCollectionItem = true;
+        return;
+      }
+      logTrackerEvent(
+        this,
+        'NFT',
+        'nft_class_details_switch_to_edition',
+        this.classId,
+        1
+      );
+      this.shouldShowCollectionItem = false;
+    },
+    handleClickCollectionHint() {
+      logTrackerEvent(
+        this,
+        'NFT',
+        'nft_class_details_click_collection_hint',
+        this.classId,
+        1
+      );
+      this.shouldShowCollectionItem = true;
+      this.scrollToElement();
+    },
+    scrollToElement() {
+      setTimeout(() => {
+        const currentRef = this.$refs.compareSection;
+        currentRef.scrollIntoView({
+          behavior: 'smooth',
+          block: 'start',
+        });
```

Comment:
Not sure if need to check `currentRef` exists

Reply:
[🎨 Update scrollToCompareSection method](https://github.com/likecoin/liker-land/pull/1516/commits/cfeba5c6882a764cff83309b83aec4a094a14833)